### DISCUSSION TITLE
email as a username

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14517] [How to]
Is there any disadvantages of using email as username? It is not allowed at the moment.
I think it common across other systems?

**Comment by mariuszkrzaczkowski** [comment]
e-mail address can be used as username, it is not blocked
![image](https://user-images.githubusercontent.com/10198654/113973426-9d553980-983c-11eb-9794-20cbcd7a7c6e.png)
